### ISSUE TITLE
Solar arrays probably need some balancing

### ISSUE THREAD

**Issue by Regularitee** [CleverRaven/Cataclysm-DDA#20882]
Currently solar arrays have 250 HP, and receive **10%** damage. Compare this to standard vehicle boards which have 240 HP, and receive **100%** damage. Meaning solar arrays are functionally as tough as ten vehicle boards.

On top of that, because they can replace vehicle quarterpanels as a defense, they effectively take up no actual vehicle space (i.e., you can replace vehicle quarterpanels with solar arrays, resulting in no increase in vehicle size). You can also combine them with curtains to act as vehicle boards, if you want to block line of sight as well.

And that's to say nothing of their first and foremost purpose of generating power. Since arrays have 1200% the generation of normal solar panels, a vehicle using these instead of boards and quarterpanels will effectively have infinite power with no fuel consumption.

Currently, there are only two real drawbacks: The high initial cost (but no upkeep costs, as they can be repaired with only a welder, which they themselves generate power for), and high weight. But even the weight isn't a notable drawback in the case of heavy war vehicles or mobile bases, which already weigh so much that the additional weight isn't noteworthy.

**So to summarize:** Solar arrays have outstanding defense, effectively no volume, produce massive amounts of free power and allow for fullness vehicles. They seem too good in to many different ways to not be nerfed. 

You guys have any thoughts? Personally, I think they should be made much more fragile so that they can only be used inside enclosed vehicles, thus actually using up vehicle volume. Maybe even reducing their power-to-volume ratio by reducing power generation from 1200% to maybe 600% or 400%?. Right now even two or three of them can generate enough power to move a heavy armored vehicle almost perpetually.

**Comment by Coolthulhu** [comment]
The damage thing isn't multiplier on damage taken, but rather damage dealt in collision.
That is, if you ram a solar panel into something, it will take 10% the damage it would if you rammed it with something with 100% damage multiplier.

Solar arrays are a mod thing, most likely blazemod.

**Comment by Regularitee** [comment]
Ah, okay. Since this is a mod issue, I guess it doesn't belong here. I'll close it down unless there are any objections.
